test/visibility/import/edgeCases/multipleDef2.good

```diff
@@ -0,0 +1,4 @@
+multipleDef2.chpl:2: error: symbol x is multiply defined
+multipleDef2.chpl:5: note: also defined here
+multipleDef2.chpl:14: In function 'main':
+multipleDef2.chpl:15: error: Symbol 'x' undeclared in module 'C'
```

Comment:
Yeah, I don't really like the multiple errors, the presence of second one seems like it could confuse the user

Reply:
Is the additional error for multipleDef3 indicating that we run down the same path multiple times?

Reply:
I think not: We run through `public import`s in `C` and see that there are conflicting symbols, give first two lines of error, don't return any one of the symbols. As a result `C.x` doesn't resolve, and we give the other two lines of error.

Reply:
Among the multiple error cases, I think I like this one and dislike the other one. But wondering what you think as well

Reply:
I don't really like either, since technically there is a symbol 'x' defined in C, just multiply defined.  But I definitely like this better than the other one